lib/generators/decidim/app_generator.rb

```diff
@@ -75,11 +74,98 @@ def secret_token
         SecureRandom.hex(64)
       end
 
+      def bundle_install
+        Bundler.with_clean_env { run "bundle install" }
+      end
+
       def install
-        Decidim::Generators::InstallGenerator.start [
```

Comment:
I think this was done so you could install decidim on an already existing app, so I'm not sure about this commit 😕